executor/write.go

```diff
@@ -881,6 +883,7 @@ func (e *InsertExec) exec(goCtx goctx.Context, rows [][]types.Datum) (Row, error
 			continue
 		}
 		if batchInsert && rowCount >= batchSize {
+			e.ctx.Txn().FlushStatement(true)
```

Comment:
How about call e.ctx.Txn().FlushStatement(true) in `e.ctx.NewTxn()`? Avoid missing Flush.

Reply:
Not a good ides.
The statement itself should be responsible for its whole life span, from the beginning to flush or discard. If it forgot to do so, `e.ctx.NewTxn()` could not remedy for that:
 is it  `FlushStatement(true)` or `FlushStatement(false)` ?